core/src/main/java/com/google/instrumentation/trace/TraceOptions.java

```diff
@@ -25,54 +29,83 @@
  */
 @Immutable
 public final class TraceOptions {
+  // Default options. Nothing set.
+  private static final int DEFAULT_OPTIONS = 0;
+  // Mask to extract a byte value.
+  private static final int BYTE_MASK = 0xFF;
+  // Bit to represent whether trace is sampled or not.
+  private static final int IS_SAMPLED = 0x1;
+
   /**
-   * Bit to represent whether trace is sampled or not. If set, means this {@code Span} is part of a
-   * sampled trace and data should be exported to a persistent store.
+   * The size in bytes of the {@code TraceOptions}.
    */
-  public static final int IS_SAMPLED = 0x1;
+  public static final int SIZE = 4;
 
-  private static final int DEFAULT_OPTIONS = 0;
-
-  private static final TraceOptions DEFAULT_TRACE_OPTIONS = new TraceOptions(DEFAULT_OPTIONS);
+  /**
+   * The default {@code TraceOptions}.
+   */
+  public static final TraceOptions DEFAULT = new TraceOptions(DEFAULT_OPTIONS);
 
   // The set of enabled features is determined by all the enabled bits.
   private final int options;
 
+  // Creates a new {@code TraceOptions} with the given options.
+  private TraceOptions(int options) {
+    this.options = options;
+  }
+
   /**
-   * Returns the default {@code TraceOptions}: Sampling disabled.
+   * Returns a {@code TraceOptions} whose representation is given param.
```

Comment:
Better would be:

* Returns a {@code TraceOptions} built from a byte representation.